### ISSUE TITLE
Convert "Failed to extend session" toast to alert

### ISSUE THREAD

**Issue by myfriendgu** [civiform/civiform#13697]
#### Describe the solution presented in this issue.

We are trying to convert all toast messages to alerts. This ticket is for the following toast message: 

"Failed to extend session": Red error toast. Auto-dismisses after 3 seconds. Dismissible via button. Triggered when you click "Extend Session" in the inactivity warning modal and there's a network error in extending the session.

#### Additional Context

There are a couple scenarios for this. Test results for how this behaves are as follows:

- If they’re a guest and log in after extending the session fails:
    - They are kept on the same page after logging in
    - Their data is retained
    - The act of logging in extends their session because they are no longer inactive
- If they're a guest and create a new account after extending the session fails:
    - Same behavior as above

See [OG ticket](https://github.com/civiform/civiform/issues/7749#issuecomment-5038203729) for previous discussion

#### Done when:

What criteria need to be met for the issue to be finished?

- [ ] Design page that displays an error alert
- [ ] Copy that describes that the session failed to extend, but that their data is retained if they log in or create an account with link to the sign in page
